client/galaxy/scripts/apps/panels/tool-panel.js

```diff
@@ -58,10 +59,26 @@ var ToolPanel = Backbone.View.extend({
             href    : 'workflow'
         }));
         _.each( this.stored_workflow_menu_entries, function( menu_entry ){
-            self.$( '#internal-workflows' ).append( self._templateTool({
+            self.$( '#internal-workflows' ).append( self._templateWorkflowLink({
                 title : menu_entry.stored_workflow.name,
-                href  : 'workflow/run?id=' + menu_entry.encoded_stored_workflow_id
+                href  : 'workflow/run?id=' + menu_entry.encoded_stored_workflow_id,
+                cls   : 'workflow-menu-' + menu_entry.encoded_stored_workflow_id
             }));
+            self._registerWorkflowMenuClick( self, menu_entry.encoded_stored_workflow_id );
+        });
+    },
+
+    /** Open the items in workflow menu in the center panel */
+    _registerWorkflowMenuClick: function( self, workflow_id ) {
+        $( '.workflow-menu-' + workflow_id ).click(function( e ) {
+            var url = Galaxy.root + 'workflow/run_workflow?id=' + workflow_id;
+            $.getJSON(url, function( response ) {
```

Comment:
Using 'Utils.get' can reduce the overhead, making stringify and parse below unnecessary.